src/shelly_hap_light_bulb.cpp

```diff
@@ -204,6 +220,23 @@ void LightBulb::HSVtoRGBW(RGBW &rgbw) const {
   }
 }
 
+void LightBulb::ColortemperaturetoRGBW(RGBW &rgbw) const {
+  float v = cfg_->brightness / 100.0f;
+
+  // brightness and colortemperature in mired to cw, ww values
+  int temp_max = 400;
+  int temp_min = 50;
```

Comment:
This should be configurable via web UI, not every stripe has the same values.

Reply:
I agree, it should be in the future